api/config/v1alpha1/envoygateway_types.go

```diff
@@ -80,23 +80,23 @@ type Provider struct {
 	// provides runtime configuration via the Kubernetes API.
 	//
 	// +optional
-	Kubernetes *KubernetesProvider `json:"kubernetes,omitempty"`
+	Kubernetes *EnvoyGatewayKubernetesProvider `json:"kubernetes,omitempty"`
 
 	// File defines the configuration of the File provider. File provides runtime
 	// configuration defined by one or more files. This type is not implemented
 	// until https://github.com/envoyproxy/gateway/issues/1001 is fixed.
 	//
 	// +optional
-	File *FileProvider `json:"file,omitempty"`
+	File *GatewayFileProvider `json:"file,omitempty"`
```

Comment:
`EnvoyGateway` prefix here as well, to make it all consistent ?

Reply:
found it ,fixed